### PR TITLE
Fix Ninor Errors in AI Files

### PR DESCRIPTION
**Pull request by vmelamed**
# Summary

## Commits

- fix: minor errors in AI files

## Checklist

- [ ] Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/) format
- [ ] Tests pass locally, and cover new code and edge cases. Test coverage is above 80%.
- [ ] It is not expected that the performance will degrade by more than 20%
- [ ] Documentation is updated if necessary

### CODE REVIEW

File: .github/copilot-instructions.md [hunk 1]
```diff
@@ -11,15 +11,7 @@ use the same source of truth.
...
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
-
-@~/.claude/CLAUDE.md
-@~/repos/vm2/CLAUDE.md
-@.github/CONVENTIONS.md
-@docs/ARCHITECTURE.md
-@docs/WORKFLOWS_REFERENCE.md
-@docs/GIT_PLAYBOOK.md
-@docs/RELEASE_PROCESS.md
+This file provides guidance to Copilot when working with code in this repository.
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md]: fix

File: .github/copilot-instructions.md [hunk 2]
```diff
@@ -54,15 +46,15 @@ CI.yaml ───────┬───────► actions/gather-inputs/a
 
 Prerelease.yaml ───────► _prerelease.yaml ───────────────┬─► compute-prerelease-version.sh ─►
                                                          ├─► changelog-and-tag.sh ──────────►
-                                                         └─► publish-package.s ─────────────►
+                                                         └─► publish-package.sh ────────────►
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md]: fixed

File: CLAUDE.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ CI.yaml ───────┬───────► actions/gather-inputs/a
 
 Prerelease.yaml ───────► _prerelease.yaml ───────────────┬─► compute-prerelease-version.sh ─►
                                                          ├─► changelog-and-tag.sh ──────────►
-                                                         └─► publish-package.s ─────────────►
+                                                         └─► publish-package.sh ────────────►
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply CLAUDE.md]: fixed
